skycultures/greek_leidenAratea/description.en.utf8

```diff
@@ -0,0 +1,319 @@
+</p>
+<p ><br/>
+
+</p>
+<table >
+	
+	<tr valign="top">
+		<td>	<b>Latin Name, Abbreviation</b>		</td>
+		<td><b>	Original Greek Name</b>		</td>
+		<td><b>	Translation</b>			(literally)		</td>
+		<td><b>	Picture</b>		</td>
+	</tr>
+	<tr valign="top">
+		<td> Ursa Minor, Ursa Maior, Draco <br>
+			UMi, UMa, Dra		</td>
+		<td>			 &Mu;&iota;&kappa;&rho;&#942;
+			&#702;&#714;&Alpha;&rho;&kappa;&tau;&omicron;&sigmaf; , 
+			&Mu;&epsilon;&gamma;&alpha;&lambda;&#942;
+			&#702;&#714;&Alpha;&rho;&kappa;&tau;&omicron;&sigmaf; , 
+			&Delta;&rho;&#940;&kappa;&omega;&nu;
+ 			</td>
+		<td>Little She-Bear, Great She-Bear, Dragon	(Latin) / Snake (Greek)	</td>
+		<td> <img width="255" src="descr_pics/aratea_small_dra.jpg"/> </td>
+	</tr>
+	<tr valign="top">
+		<td> King Cepheus, Cep</td>
+		<td>&Kappa;&eta;&phi;&epsilon;&#973;&sigmaf; </td>
+		<td> Kepheus</td>
+		<td> <img width="255" src="descr_pics/aratea_small_cep.jpg"/> </td>
+	</tr>
+	<tr valign="top">
+		<td>Bootes Boo	</td>
+		<td> &Beta;&omicron;&#974;&tau;&eta;&sigmaf; </td>
+		<td>Ox driver (Latin), Ploughing Man (Greek) </td>
+		<td> <img width="255" src="descr_pics/aratea_small_boo.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Northern Crown, CrB 	</td>
+		<td>&Sigma;&tau;&#941;&phi;&alpha;&nu;&omicron;&sigmaf; </td>
+		<td>Crown (Latin), Wreath (Greek)</td>
+		<td> <img width="255" src="descr_pics/aratea_small_crB.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Hercules, Her</td>
+		<td>&#700;&Epsilon;&nu;&gamma;&#972;&nu;&alpha;&sigma;&iota;&nu;</td>
+		<td>Hercules (Latin), Kneeler (Greek) <br> The star between the legs is a well-known mistake in the picture: for this figure the draughtsman obviously took Bootes as a model; leg position, shepherd's crook (instead of club for Hercules) and star between the knees are the same.</td>
+		<td> <img width="255" src="descr_pics/aratea_small_her.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Lyra, Lyr</td>
+		<td>&Lambda;&#973;&rho;&alpha; </td>
+		<td>Lyre</td>
+		<td> <img width="255" src="descr_pics/aratea_small_lyr.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Cygnus, Cyg </td>
+		<td>&#700;&#714;&Omicron;&rho;&nu;&iota;&sigmaf; 	</td>
+		<td>Bird [not specified] </td>
+		<td> <img width="255" src="descr_pics/aratea_small_cyg.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Queen Cassiopeia, Cas</td>
+		<td>&Kappa;&alpha;&sigma;&sigma;&iota;&#941;&pi;&epsilon;&iota;&alpha; </td>
+		<td>Queen Kassiepeia </td>
+		<td> <img width="255" src="descr_pics/aratea_small_cas.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Perseus, Per	</td>
+		<td>&Pi;&epsilon;&rho;&sigma;&epsilon;&#973;&sigmaf;</td>
+		<td>Perseus <br> mirrored depiction</td>
+		<td> <img width="255" src="descr_pics/aratea_small_per.jpg"/> 		</td>
+	</tr>
+	<tr valign="top">
+		<td>Auriga, Aur		</td>
+		<td>&#699;&Eta;&nu;&#943;&omicron;&chi;&omicron;&sigmaf; </td>
```

Comment:
I am pretty sure also here and in several more places (all initial vowels where there is no sp. asper...) we should have spiritus lenis &#702; (702), not this accent &#699; (699). Please double- and triple-check both lists.